geom/type_line_string.go

```diff
@@ -32,6 +32,7 @@ func NewLineString(seq Sequence, opts ...ConstructorOption) (LineString, error)
 	// Valid non-empty LineStrings must have at least 2 *distinct* points.
 	if !hasAtLeast2DistinctPointsInSeq(seq) {
 		if ctorOpts.omitInvalid {
+			// TODO: Should the coordinates type match the input seq?
```

Comment:
For my understanding, is this TODO relating to the empty `LineString{}` being returned?

Reply:
Yes, that's correct. The TODO is not related to this PR at all, so I'll create a new GH issue and remove the TODO.

Reply:
I've created a new issue to track this problem: https://github.com/peterstace/simplefeatures/issues/432

I've also removed the TODO in ee74938511f5b9a047318b61a19b68fc1ea78b0e (since it's tracked by an issue, the TODO just clutters things up).